ansible/templates/whisk.properties.j2

```diff
@@ -52,4 +52,4 @@ python.27=python
 use.cli.download=false
 nginx.conf.dir={{ nginx_conf_dir }}
 testing.auth={{ openwhisk_home }}/config/keys/auth.guest
-vcap.services.file=
+vcap.services.file={{openwhisk_home}}/tests/credentials/vcap_services.json
```

Comment:
Also should we use a different term rather than vcap per @mbehrendt's suggestion?